src/index.html

```diff
@@ -3,8 +3,17 @@
 <head>
     <meta charset="UTF-8">
     <title>Country Information - World Regions</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+<h1 id="earth">Planet Earth!</h1>
 
+<img id="worldmap" src="assets/world_map.png" alt="Wereldmap">
+
+<h1 id="countries-title">All the countries in the world:</h1>
+<div id="container">
+    <ul id="country-list"></ul>
+</div>
+<script type="module" src="app.js"></script>
 </body>
```

Comment:
He... een dejavu... Deze file heb ik al gezien, al is dit de originele uit de src folder waar het om gaat